README.md

```diff
@@ -34,7 +34,7 @@ To get a local copy up and running follow these simple example steps.
 - Click on the button at the top left to diplay the header
 - Click in home to go to the list
 - Click About to see the About page
-- Write a comment in the input box and hit submit to create a new list item
+- Write a comment in the input box and hit 'Enter' to create a new list item
 - Click the Delete button to delete an item
 
 ### Run tests
```

Comment:
I would suggest being more descriptive when creating the README file. Adding a screenshot and deleting the sections that are not in use :technologist:

Reply:
Readme has been fixed